src/backend/joanie/core/serializers/client.py

```diff
@@ -736,6 +736,46 @@ class Meta:
         read_only_fields = fields
 
 
+class NestedCourseOrderSerializer(serializers.ModelSerializer):
+    """Order model serializer for the Course model"""
+
+    id = serializers.CharField(read_only=True)
+    learner_username = serializers.CharField(source="owner.username", read_only=True)
+    learner_email = serializers.CharField(source="owner.email", read_only=True)
+    learner_full_name = serializers.CharField(
+        source="owner.get_full_name", read_only=True
+    )
```

Comment:
IMO this should be in a `owner` property, using a `UserLite` serializer containing only what we expect.